src/common/MovieTile/styled.js

```diff
@@ -0,0 +1,87 @@
+import { styled } from "styled-components";
+
+export const Wrapper = styled.div`
+  width: 324px;
+  height: 650px;
+  border-radius: 5px;
+  box-shadow: 0px 4px 12px 0px rgba(186, 199, 213, 0.5);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
I think you could make a separate div to wrap only Description and Rating and use justify-content: space-between there instead

Reply:
I added 2 div-s and that did the trick